src/characters/characters.controller.ts

```diff
@@ -34,7 +46,11 @@ export class CharactersController {
   @ApiBody({ type: CharactersDto })
   @Put(':id')
   @UseGuards(AuthGuard('jwt'))
+<<<<<<< HEAD
```

Comment:
will those head markers be pushed?

Reply:
Not sure, most likely not, but that is not part of my code it is from Saras.